### PR TITLE
Use acked task in UpdateSubscriptionAction and repeat test_schema_changes_of_subscribed_table_is_replicated_and_new_data_is_synced

### PR DESCRIPTION
**Pull request by BaurzhanSakhariev**

### CODE REVIEW

**Comment by BaurzhanSakhariev** [comment]
Failed runs on this PR before increasing `assertBusy` timeout:

https://github.com/crate/crate/actions/runs/6544421167/job/17770850029?pr=14902

https://jenkins.crate.io/job/CrateDB/job/crate_on_pr/10723/testReport/junit/io.crate.integrationtests/MetadataTrackerITest/___/

File: server/src/test/java/io/crate/integrationtests/MetadataTrackerITest.java [hunk 1]
```diff
@@ -62,6 +63,7 @@ public void test_schema_changes_of_subscribed_table_is_replicated() throws Excep
     }
 
     @Test
+    @Repeat(iterations = 500)
```
Comment (BaurzhanSakhariev) [inline server/src/test/java/io/crate/integrationtests/MetadataTrackerITest.java:66]: Will remove this

**Comment by mfussenegger** [comment]
the gap in duration between successful run and failed run are quite high. Are you sure this is hitting the timeout too early?

**Comment by BaurzhanSakhariev** [comment]
> the gap in duration between successful run and failed run are quite high. Are you sure this is hitting the timeout too early?

You mean that if in average 5 seconds is enough to replicate all schema changes, then we need to figure out why have some spikes? 

I will have another look. So far, increased timeout helps which means mechanism itself is working but sometimes has a huge delay. I will look what could be the root cause of that occasional replication speed drops.

**Comment by mfussenegger** [comment]
> I will have another look. So far, increased timeout helps which means mechanism itself is working but sometimes has a huge delay

In that case we could also merge the timeout increase to get the CI stable and continue investigating the huge delay in

**Comment by BaurzhanSakhariev** [comment]
> In that case we could also merge the timeout increase to get the CI stable and continue investigating the huge delay

It passed locally but on CI even 50 seconds doesn't work. We cannot merge it, there is some issue with schema changes being not visible, will investigate further.

**Pull request closed without merging**